man/bootup.xml

```diff
@@ -23,17 +23,25 @@
   <refsect1>
     <title>Description</title>
 
-    <para>A number of different components are involved in the system
-    boot. Immediately after power-up, the system BIOS will do minimal
-    hardware initialization, and hand control over to a boot loader
-    stored on a persistent storage device. This boot loader will then
-    invoke an OS kernel from disk (or the network). In the Linux case,
-    this kernel (optionally) extracts and executes an initial RAM disk
-    image (initrd), such as generated by
+    <para>A number of different components are involved in the boot of a Linux system. Immediately after
+    power-up, the system BIOS will do minimal hardware initialization, and hand control over to a boot loader
```

Comment:
if you touch that, we probably shouldn't call this "system BIOS" anymore, but "system firmware"...